static/translations/messages_tl.json

```diff
@@ -156,8 +156,13 @@
   "{realm} is running Zulip Server {serverVersion}, which is unsupported. Please contact your administrator about upgrading.": "{realm} is running Zulip Server {serverVersion}, which is unsupported. Please contact your administrator about upgrading.",
   "Learn more": "Learn more",
   "Full profile": "Full profile",
+<<<<<<< HEAD
+  "Settings": "Settings",
+  "Night mode": "Night mode",
+=======
   "Settings": "Mga Itinakda",
   "Night mode": "Night mode",
+>>>>>>> c8af6d69ecbed660643facc1e84b416e3dbff879
```

Comment:
It looks like you accidentally left this in; please amend your commit to remove this change.